src/applications/travel-pay/containers/TravelPayStatusApp.jsx

```diff
@@ -180,6 +180,41 @@ export default function App({ children }) {
     displayedClaims = displayedClaims.slice(pageStart - 1, pageEnd);
   }
 
+  const resultsText = () => {
+    let sortAndFilterText;
+
+    if (orderClaimsBy === 'mostRecent') {
+      sortAndFilterText = 'date (most recent)';
+    }
+
+    if (orderClaimsBy === 'oldest') {
+      sortAndFilterText = 'date (oldest)';
+    }
+
+    let appliedFiltersLength = appliedStatusFilters.length;
+    if (appliedDateFilter !== 'all') {
+      appliedFiltersLength += 1;
+    }
+
+    if (appliedFiltersLength > 0) {
+      sortAndFilterText += `, with ${appliedFiltersLength} ${
+        appliedFiltersLength === 1 ? 'filter' : 'filters'
+      } applied`;
+    }
+
+    if (numResults === 0 && appliedFiltersLength > 0) {
+      // Note that appliedFiltersLength === 1 should never be true here, since
+      // 0 claims matching a single filter should mean the filter isn't shown
```

Comment:
`appliedFiltersLength == 1` is possible if one of the hardcoded status filters was applied, and it's possible that none of the claims match the hardcoded status

Reply:
If I understand what you're referring to, hardcoded filters got removed in https://github.com/department-of-veterans-affairs/vets-website/pull/30877 and made `vets-api`/the upstream travel pay API the source of truth of status filter names, which I think means this comment is accurate? I could be missing something, though.

Reply:
By hardcoded filters, I meant the statuses we always show in the filters no matter what. These statuses are: `'On Hold', 'Denied', 'In Manual Review'`
So for example, it's possible for none of the claims to be `In Manual Review`, but the `In Manual Review` filter would still be displayed.

Reply:
I see what you're saying now. While we _are_ hardcoding the top statuses, [we're still filtering them out](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/travel-pay/containers/TravelPayStatusApp.jsx#L56-L62) if they're not present in any of the returned claims